### DISCUSSION TITLE
Camera View (jsmpeg mse webrtc)

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#5255] [General]
Just wondering what determines the drop down option. Some of my cameras only show "jsmpeg" while others show all 3 options. Cameras are different model. See image.
![DropDown](https://user-images.githubusercontent.com/59022856/214859541-7e0b14e4-db50-448d-826d-80f87364aab5.jpg)

**Comment by NickM-27** [comment]
You need to either have the names matching in go2rtc and frigate config (case sensitive) or you need to set `live -> stream_name`

**Reply by 18923** [reply]
Case sensitive YoU SaY... Thank you that fixed it.

**Comment by adriel** [comment]
Is this something you have add to Frigate?
I only get `jsmpeg`, I only have 1 camera. Is there a benefit to using one of the others? (assuming I can get them)

**Reply by NickM-27** [reply]
You just need to follow the docs https://deploy-preview-4055--frigate-docs.netlify.app/guides/configuring_go2rtc

MSE / webRTC offer full resolution, full frame rate, include audio, and have less latency than jsmpeg.

**Reply by adriel** [reply]
aaaah, legend, thank!

**Comment by bartbutenaers** [comment]
Remark for novice users - like me - that can find nowhere the dropdown from the above screenshot.  It [seems](https://github.com/blakeblackshear/frigate/discussions/12432) that this dropdown has been removed some time ago:

+ Version ***0.13***: a dropdown menu in the UI allowed users to select between jsmpeg, MSE, or WebRTC. 
+ Version ***0.14***: Frigate [intelligently ](https://www.reddit.com/r/frigate_nvr/comments/1dz0uk0/comment/lce4r5d/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)selects the best type of live view method for the cameras on your dashboard, so this is _**no**_ longer user selectable.
